torch/_tensor_docs.py

```diff
@@ -1478,6 +1478,20 @@ def add_docstr_all(method, docstr):
 See :func:`torch.nn.functional.hardshrink`
 """)
 
+add_docstr_all('heaviside',
+               r"""
+heaviside(val) -> Tensor
```

Comment:
`heaviside(other)` would probably be more consistent with our naming

Reply:
What about `heaviside(values)` , just like the parameter `values (Tensor or Scalar)` in `torch.searchsorted()`?

Reply:
OK, that makes sense.